### PR TITLE
Correcting some merging errors and some unit testing

### PR DESCRIPTION
**Pull request by nicholasyung1**
Some duplicate declarations and missing brackets.

### CODE REVIEW

**Review by Zhangjason308 — COMMENTED** [review]
I think you should be able to close this PR, my current PR removes the merge conflicts and also displays the Cards now.

File: server/src/test/java/MembershipTest.java [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+import com.example.model.Perk;
+import com.example.model.Membership;
+import com.example.model.User;
 
 import java.util.ArrayList;
 
 class MembershipTest {
 
+    @Test
+    public void MembershipTest(){
```
Comment (Zhangjason308) [inline server/src/test/java/MembershipTest.java:12]: can you move this into the test/java/com/example/model package

and make sure they are packaged `package com.example.model;
`

File: server/src/main/java/com/example/model/Membership.java [hunk 2]
```diff
@@ -18,6 +18,7 @@ public class Membership {
     @OneToMany(mappedBy = "membership", cascade = CascadeType.ALL, orphanRemoval = true)
     private List<Perk> perks = new ArrayList<>();
 
+
```
Comment (Waluiyeegi) [inline server/src/main/java/com/example/model/Membership.java:21]: Unneccesary space, usually 1 space is enough

File: server/src/test/java/PerkTest.java [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
 import org.junit.jupiter.api.Test;
-
+import com.example.model.Perk;
 import java.util.ArrayList;
 
 class PerkTest {
 
+    @Test
+    public void PerkTest(){
+        Perk perk = new Perk();
```
Comment (Zhangjason308) [inline server/src/test/java/PerkTest.java:10]: Make sure to test all properties such as the Id, upvote, downvotes, etc. so you can test the methods

File: server/src/test/java/ProductTest.java [hunk 4]
```diff
@@ -1,6 +1,19 @@
 import static org.junit.jupiter.api.Assertions.*;
+
+import com.example.model.Product;
 import org.junit.jupiter.api.Test;
 
 class ProductTest {
 
+    @Test
+    public void productTest() {
+
```
Comment (Zhangjason308) [inline server/src/test/java/ProductTest.java:10]: Same here

**Review by Zhangjason308 — CHANGES_REQUESTED** [review]
Make sure your unit test cases have full coverage

File: server/src/test/java/com/example/model/PerkTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        perk.setName(name);
+        perk.setMembership(membership);
+        perk.downvote();
+        perk.upvote();
+        perk.setCode(code);
+        perk.setGeographicArea(geoArea);
+        perk.setProduct(product);
+        perk.setExpiryDate(expiryDate);
+
+        assertEquals(perk.getDescription(), description);
```
Comment (Zhangjason308) [inline server/src/test/java/com/example/model/PerkTest.java:34]: looks good

**Review by Zhangjason308 — APPROVED** [review]

File: server/src/test/java/MembershipTest.java (hunk 1 above)
Comment (Zhangjason308) [reply server/src/test/java/MembershipTest.java:12]: still need to move the membershiptest and product test to the correct files

File: server/src/test/java/com/example/model/ProductTest.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
+package com.example.model;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.example.model.Product;
+import org.junit.jupiter.api.Test;
+
+class ProductTest {
+
```
Comment (Zhangjason308) [inline server/src/test/java/com/example/model/ProductTest.java:9]: looks good

**Review by Zhangjason308 — APPROVED** [review]

**Pull request merged**